app/models/user.rb

```diff
@@ -57,7 +57,13 @@ class User < ActiveRecord::Base
 
   validates :top_3_interests, length: { maximum: 3, too_long: " is limited to %{count} interests" }
   validates_presence_of :first_name, :last_name
-  validates_presence_of :mentor_industry, if: :mentor
+
+  has_many :mentor_industry_users, dependent: :destroy
+  has_many :mentor_industries, through: :mentor_industry_users
+  accepts_nested_attributes_for :mentor_industry_users, allow_destroy: true
+
+  # validates_presence_of :mentor_industry, if: :mentor
```

Comment:
I'd like to remove all comments. https://github.com/bbatsov/ruby-style-guide#comments